doc/src/fix_brownian.rst

```diff
@@ -0,0 +1,151 @@
+.. index:: fix brownian
+
+fix brownian command
+====================
+
+Syntax
+""""""
+
+.. parsed-literal::
+
+   fix ID group-ID brownian gamma_t diff_t seed keyword args
+
+* ID, group-ID are documented in :doc:`fix <fix>` command
+* brownian/sphere = style name of this fix command
+* gamma_t = translational friction coefficient
+* diff_t = translational diffusion coefficient
+* zero or more keyword/value pairs may be appended
+* keyword = *rng* 
+
+  .. parsed-literal::
+
+        *rng* value = *uniform* or *gaussian* or *none*
+         *uniform* = use uniform random number generator
+         *gaussian* = use gaussian random number generator
+         *none* = turn off noise
+
+Examples
+""""""""
+
+.. code-block:: LAMMPS
+
+   fix 1 all brownian 1.0 3.0 1294019
+   fix 1 all brownian 1.0 3.0 19581092 rng none 
+   fix 1 all brownian 1.0 3.0 19581092 rng uniform
+   fix 1 all brownian 1.0 3.0 19581092 rng gaussian
+
+
+Description
+"""""""""""
+
+Perform Brownian Dynamics integration to update position and velocity
+of atoms in the group each timestep. Brownian Dynamics uses Newton's laws of
+motion in the limit that inertial forces are negligible compared to
+viscous forces. The stochastic equation of motion is
+
+.. math::
+
+   dr = \frac{F}{\gamma_t}dt+\sqrt{2D_t}dW_t, \\
+
+where :math:`dW_t` is a Wiener processes (see e.g. :ref:`(Gardiner) <GardinerC1>`).
+
+.. note::
+   This integrator is designed for generic non-equilibrium
+   simulations with additive noise. There are two important cases which
+   (conceptually) reduce the number of free parameters in this fix.
+   (a) In equilibrium simulations
+   (where fluctuation dissipation theorems are obeyed), one can define
+   the thermal energy :math:`k_bT=D_t\gamma_t`.
+   
+---------
+
+.. note::
+   Temperature computation using the :doc:`compute temp <compute_temp>`
```

Comment:
should we then perhaps have a new temperature compute included in this package that outputs a temperature which is commensurate with the brownian dynamics conditions? most thermostat fixes create a temperature compute internally and offload the computation of the system temperature to that compute and also have an option to modify this or even do some kind of "biasing", where the thermostat can be restricted to what is thermostatted how, e.g. to dimensions, regions or gradients. Note, I am not says that this must be done, but am merely suggesting to think about this. Check out the HowTo document about thermostatting (and the included links) in the manual for some general information. This could also be done in a second step as a separate pull request. This one looks mostly OK, so I would prefer to merge this ASAP and do further improvements later rather than postponing the merge indefinitely until  it is "better than perfect".

Mind you, that for atom style sphere, you would need to compare to compute temp/sphere and for atom style ellipsoid to compute temp/asphere anyway, as compute temp does not account for the rotational kinetic energy.

Reply:
Would be interested in doing this in the future, but figured given how long I've dragged my feet on this its better to get this out now and then start a new PR as you've said.